src/extension/features/budget/subtract-upcoming-from-available/settings.js

```diff
@@ -6,15 +6,13 @@ module.exports = {
   title: 'Subtract Upcoming Transactions from Available Balance',
   description:
     'Subtracts upcoming transactions from the available balance for each category.' +
-    ' In other words, treat upcoming transactions as if the money has already been spent. Also adds "Available After Upcoming Transactions" to the budget breakdown.' +
-    '\n\nEnabling "Subtract Credit Card Balances from Total Available" will total the amounts in the "Payment" column' +
-    ' of your CC category group and subtract that from the "Available After Upcoming Transactions" in the budget breakdown.' +
-    ' This allows to see how much you have available, excluding the money "reserved" in your Credit Card Payments category group.' +
-    ' (Probably only useful if you pay off your CCs in full every month.)',
+    ' In other words, treat upcoming transactions as if the money has already been spent. Also shows "Available After Upcoming Transactions" in the budget breakdown.' +
+    '\n\nAdditionally, this feature totals the amounts in the "Payment" column of your CC category group and subtracts that from the "Available After Upcoming Transactions" in the budget breakdown.' +
+    ' This allows you to see how much you have available if you exclude the money "reserved" in your Credit Card Payments category group.' +
+    ' You can turn this part of the feature off by selecting "Don\'t Include CC Payments".',
   options: [
     { name: 'Disabled', value: '0' },
```

Comment:
once rebased on `live-settings`, you'll want to make `default: false` and remove this option